GitUI/Editor/FileViewer.cs

```diff
@@ -1539,24 +1539,43 @@ private void ApplySelectedLines(bool allFile, bool reverse)
             }
 
             byte[]? patch;
-            if (reverse)
+
+            Validates.NotNull(_viewItem);
+            if (_viewItem.Item.IsNew)
```

Comment:
Can this be aligned with ResetNoncommittedSelectedLines etc, almost the same logic?